packages/components/src/assets/index.ts

```diff
@@ -1,3 +1,6 @@
+// Payment Flags Icons
+export * from './PaymentFlags'
```

Comment:
Fan, I think we should explain on the doc how the dev could add a new icon as PaymentFlags

Reply:
oh yeah! Lemme add.